### ISSUE TITLE
Components properties not showing up

### ISSUE THREAD

**Issue by GerardRodes** [vuejs/vue-devtools#527]
### Version
4.0.0

### Browser and OS info
Chromium Version 63.0.3239.84 (Official Build) Built on Ubuntu , running on Ubuntu 16.04 (64-bit)

### Steps to reproduce
Open vue devtools and click on a component

### What is expected?
Show the component properties on the right side

### What is actually happening?
No information is shown

---
Devtool is returning and error, here is what I found:
![screenshot from 2018-01-17 11-23-08](https://user-images.githubusercontent.com/16310380/35037854-07a5edb6-fb79-11e7-80ec-cb4e2d1794e0.png)
![screenshot from 2018-01-17 11-22-41](https://user-images.githubusercontent.com/16310380/35037857-0a3f6e9e-fb79-11e7-8148-713b03e18f1c.png)
![screenshot from 2018-01-17 11-23-58](https://user-images.githubusercontent.com/16310380/35037858-0a5c6bb6-fb79-11e7-9890-70407930efc1.png)

Btw, the version is 4.0.1 actually, but can't select it on the Issue Helper...

**Comment by theaarontaylor** [comment]
Not working for me as well

**Comment by SimonThordal** [comment]
Same here on firefox 57.0.1 (64-bit)
When clicking a component I get

```
TypeError: o is null[Learn More] backend.js:1:18633
	x/< moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:18633
	x moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:18295
	[6]/t.a moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:14243
	y moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:1916
	e moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5817
	e moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5959
	e moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5837
	e moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5837
	[12]/t.stringifyStrict moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5645TypeError: o is null[Learn More] backend.js:1:18633
	x/< moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:18633
	x moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:18295
	[6]/t.a moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:14243
	y moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:1916
	e moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5817
	e moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5959
	e moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5837
	e moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5837
	[12]/t.stringifyStrict moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5645
	[12]/t.stringify moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5173
	[0]/t.o moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:662
	O moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:16989
	g/< moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:15076
	[20]/n.prototype.emit moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:11449
	[19]/t.a</< moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:10452
	n moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:4719

	[12]/t.stringify moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:5173
	[0]/t.o moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:662
	O moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:16989
	g/< moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:15076
	[20]/n.prototype.emit moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:11449
	[19]/t.a</< moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:10452
	n moz-extension://8deb1d38-389a-4a17-aea6-67df9b68bd9c/build/backend.js:1:4719
```

**Comment by ixperiencenl** [comment]
Same issue here for a while already. Chrome 63.0.3239.132 (Official build) (64-bits) and Vue devtools 4.0.1 - think it has something to do with the latest update.

**Comment by icaioincau** [comment]
Same here, MacOS , Chrome 63.0.3239.132 (Official Build) (64-bit)

Vue 2.5.8 and DevTools 4.0.1

**Comment by dshagaa** [comment]
Same, Win10, Firefox 57.0.4 (64-bits) Vue v2.5.13 Vue DevTools 4.0.1

**Comment by dwoutsourcing** [comment]
Same here (Vue 2.5.0, Win 10 (64 bits), Firefox 57.0.4)

**Comment by dshagaa** [comment]
The origin of issue is vee-validate and has fixed, in the new version (2.0.3), only run npm update
----
[vue-devtools issues #508 ](https://github.com/vuejs/vue-devtools/issues/508)
[vee-validate issues #1111](https://github.com/baianat/vee-validate/issues/1111)

**Comment by Serubin** [comment]
@dshagaa I'm experiencing a similar issue but I'm not using vee-validate. Also experiencing #507 

It seems like it can't correctly connect to the vue instance. All the various tabs in the vue devtools are empty and say "no component" or "cannot detect vuex."
